packages/devtools/web/styles.css

```diff
@@ -2,16 +2,58 @@
 /* Use of this source code is governed by a BSD-style license that can be */
 /* found in the LICENSE file. */
 
+/* All color values that need to be customized depending on whether a light */
+/* or dark theme is used must be specified with CSS variables. */
+/* TODO(jacobr): the current list of variables is a bit redundant and should */
+/* be refactored to eliminate cases where very similar colors are used with */
+/* no clear reason to diverge. */
+/* If you add a CSS variable here, be sure to also add it to styles_dart.css */
+:root {
```

Comment:
This is the core definition of the color scheme. Some of these entries are a bit redundant. Feel free to suggest cases where slightly different colors should be merged.
Cases where the same color appear twice are mainly cases where the same color should have different versions in the dark theme and vice versa.

Reply:
super nit: can we alphabetize these?

Reply:
done. sorted here and in styles_dark.css